go.mod

```diff
@@ -299,6 +295,8 @@ replace (
 
 	// use cosmos style protobufs
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
+
+	// Use Informal Systems fork of Tendermint
 	github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.24
```

Comment:
Do we still need this?

Reply:
We need to keep the replace at minimum...gaia doesn't need tendermint directly.